documentation/modules/rn-2.4.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-24_{context}"]
+= {project-full} 2.4
+
+You can migrate virtual machines (VMs) from VMware vSphere or {rhv-full} to {virt} with {the-lc} {project-first}.
+
+The release notes describe technical changes, new features and enhancements, and known issues.
+
+[id="technical-changes-24_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+.Technical change 1
+
+Description 1
+
+.Technical change 2
+
+Description 2
+
+[id="new-features-and-enhancements-24_{context}"]
+== New features and enhancements
+
+This release has the following features and improvements:
```

Comment:
@ahadas - we tried to help you by adding what we thought were the main new features. 
However, if these are wrong, please do not hesitate to correct us. 
Appreciate it

Reply:
other features:
1. 'Skip certificate validation' option was added to VMware and RHV providers.  If checked, the provider's certificate won't be validated and the UI won't ask for specifying a CA certificate
2. When defining a RHV provider that set with third-party certificate, only the third-party certificate need to be specified and MTV would retrieve the Manager CA certificate automatically